dotcom-rendering/src/web/components/CommercialMetrics.tsx

```diff
@@ -8,11 +8,8 @@ import { spacefinderOkr1FilterNearby } from '@frontend/web/experiments/tests/spa
 import { useDocumentVisibilityState } from '../lib/useDocumentHidden';
 import { useAdBlockInUse } from '../lib/useAdBlockInUse';
 
-// TODO disallow undefined browserIds by placing conditional in App.tsx
-// so that we wait to render this component until browserId is defined.
-export const CommercialMetrics: React.FC<{
```

Comment:
Does this file `CommercialMetrics.tsx` need to be renamed to `CommercialMetrics.importable.tsx` to be the child of an `Island`?

Reply:
This is a really good point. I'm going to step back and look at if there are better ways to handle this situation as there could be higher level solutions we want to introduce.

Reply:
Yes, great spot!